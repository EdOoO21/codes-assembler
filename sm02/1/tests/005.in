-477 519 -266 628 414 930 723 515 335 888 85 -941 721 -47 589 931 -490 329 -894 845 -679 -769 -239 -40 778 -496 -221 113 -792 175 -490 -974 497 -557 -165 -428 -628 876 776 568 -203 -674 560 632 -853 -716 265 264 -89 -741 -730 -997 991 784 -990 -572 584 -559 966 -661 786 -660 -408 -358 969 -593 104 794 388 281 -581 -628 927 983 414 -597 979 976 830
