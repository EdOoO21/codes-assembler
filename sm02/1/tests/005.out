830
976
979
-597
414
983
927
-628
-581
281
388
794
104
-593
969
-358
-408
-660
786
-661
966
-559
584
-572
-990
784
991
-997
-730
-741
-89
264
265
-716
-853
632
560
-674
-203
568
776
876
-628
-428
-165
-557
497
-974
-490
175
-792
113
-221
-496
778
-40
-239
-769
-679
845
-894
329
-490
931
589
-47
721
-941
85
888
335
515
723
930
414
628
-266
519
-477
